### DISCUSSION TITLE
vuecli 5.0.4 where is webpack???

### DISCUSSION THREAD

**Discussion by a6965921** [vuejs/vue-cli#7132] [Q&A]
`const { defineConfig } = require('@vue/cli-service')
module.exports = defineConfig({
  transpileDependencies: true
})
module.exports = {
  resolve: {
    fallback: {
      fs: false,
      path: false
    }
  }
};`

wrong??

**Comment by Viggo95** [comment]
```js
// vue.config.js
module.exports = { 
  configureWebpack: {
    resolve: {
      fallback: { fs: false, path: false }
    }
  }
}

```
